sys-cluster/resource-agents/resource-agents-4.10.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="${P/resource-}"
+inherit autotools multilib
+
+DESCRIPTION="Resources pack for Heartbeat / Pacemaker"
+HOMEPAGE="http://www.linux-ha.org/wiki/Resource_Agents"
+SRC_URI="https://github.com/ClusterLabs/resource-agents/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~x86"
+IUSE="doc libnet rgmanager"
+
+RDEPEND="
+	sys-apps/iproute2
+	>=sys-cluster/cluster-glue-1.0.12-r1
+	libnet? ( net-libs/libnet:1.1 )
+"
+DEPEND="
+	${RDEPEND}
+	doc? (
+		dev-libs/libxslt
```

Comment:
These are likely both BDEPEND (Libxslt provides binaries too).

Reply:
(Didn't mean to approve!)

Reply:
That PR is NOT ready yet, it's just to track my updates